server/src/main/java/org/elasticsearch/search/aggregations/bucket/geogrid/GeoTileUtils.java

```diff
@@ -35,7 +35,11 @@ public final class GeoTileUtils {
 
     private GeoTileUtils() {}
 
-    private static final double PI_DIV_2 = Math.PI / 2;
+    private static final double PI_DIV_2 = Math.PI / 2.0;
+
+    private static final double PI_TIMES_2 = Math.PI * 2.0;
```

Comment:
I would have expected the compiler to do this for you already, but no harm in being certain about it.